packages/genesys-spark/src/hosts.ts

```diff
@@ -37,7 +37,7 @@ export function getChartComponentAssetsOrigin(): string {
   if (IS_DEV_MODE == true) {
     // This conditional is optimized out in production due to @rollup/plugin-replace
     // and rollup's dead code elimination
-    return 'http://localhost:3734';
+    return `http://${window.location.hostname}:3734`;
```

Comment:
This isn't related to accessibility but I've been testing some stuff through Parallels on a Windows machine and this makes port forwarding easier.